app/controllers/test/saml_test_controller.rb

```diff
@@ -111,5 +111,13 @@ def test_saml_settings
       settings.idp_slo_target_url = "http://#{Figaro.env.domain_name}/api/saml/logout"
       settings
     end
+
+    def saml_client_private_key
+      @saml_client_private_key ||= File.read("#{Rails.root}/config/saml_client_private_key.enc")
```

Comment:
If these files are only used for specs, then perhaps they should be moved to "spec/config"?